extension-ui/manifest.json

```diff
@@ -21,6 +21,6 @@
     "default_popup": "popup.html"
   },
    "web_accessible_resources": 
-   [{"resources": ["oops.html"],
+   [{"resources": ["oops.html","client-side/dist/index.html"],
```

Comment:
Does this change supposed to be here after you moved back the client-side folder?
if not, please slash it